core/src/main/java/greencity/mapping/HabitStatusDtoMapper.java

```diff
@@ -27,11 +28,11 @@ protected HabitStatusDto convert(HabitStatus habitStatus) {
             .lastEnrollmentDate(habitStatus.getLastEnrollmentDate())
             .workingDays(habitStatus.getWorkingDays())
             .habitAssignId(habitStatus.getHabitAssign().getId())
-            .habitStatusCalendarDtos(habitStatus.getHabitStatusCalendars().stream()
+            .habitStatusCalendarDtos(new ArrayList<>(habitStatus.getHabitStatusCalendars().stream()
```

Comment:
In that case I'm getting exception in tests. And I don't now if it's even possible to fix it. It's hard for me to remove this line with adding, because I need to return this dto as a response. So I came with ArrayList alternative to List.
![Знімок екрана 2020-10-28 162029](https://user-images.githubusercontent.com/36454393/97465116-648a6d80-194a-11eb-8649-3ba2112b1a59.png)